bin/hdfs-mesos

```diff
@@ -1,5 +1,39 @@
 #!/bin/bash
 
+FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+export MESOS_HDFS_DATA_DIR=${MESOS_HDFS_DATA_DIR:-"/tmp/hdfs/data"}
```

Comment:
I pushed a readme update. You don't have to use env vars. Changing the mesos-site.xml directly works fine.